source/MaterialXRender/ShaderRenderer.h

```diff
@@ -18,6 +18,9 @@
 
 MATERIALX_NAMESPACE_BEGIN
 
+/// Shared context handle 
+using SharedContextHandle = void*;
```

Comment:
Would `RenderContextHandle` be a better name for this type?  It's really a generalization of the notion of OpenGL Context, and this object isn't always meant to be shared.